### DISCUSSION TITLE
How to Integrate SingleFile-MV3 into a Browser Extension

### DISCUSSION THREAD

**Discussion by tangxiaoqi-tangxiao** [gildas-lormeau/SingleFile-Lite#17] [Q&A]
I have configured manifest.json and background.js according to the documentation, but when I execute await extension.getPageData in content.js, it indicates that extension does not exist. I am not sure what the reason is. I am not very experienced with browser extension development and might be making some simple mistakes without realizing it.

Here is my code:
[manifest.json](https://github.com/user-attachments/files/15937818/manifest.json)
background.js：
![image](https://github.com/gildas-lormeau/SingleFile-MV3/assets/54069236/c26e5437-c2e9-44c8-9d3a-ab4065341516)
content.js：
![image](https://github.com/gildas-lormeau/SingleFile-MV3/assets/54069236/28041c36-71db-4de5-9190-45bc51b233e3)

**Comment by gildas-lormeau** [comment]
Here's a complete example in the ZIP file.

[Uploading SingleFile-Mini-MV3.zip…]()
